lamin_cli/_save.py

```diff
@@ -42,18 +45,15 @@ def parse_uid_from_code(content: str, suffix: str) -> str | None:
     return uid
 
 
-def save_from_filepath_cli(
-    filepath: str | Path,
+def save_from_path_cli(
+    objpath: UPath,
```

Comment:
Can we change the name of this variable: `objpath`?

It's both a cryptic abbreviation but even if it was `object_path` or `objectpath` then it'd be inaccurate because it can point to a folder.

Why not just call it `path`?

Reply:
Sure, will do.